app/src/main/java/com/google/droidjump/LevelsAdapter.java

```diff
@@ -50,7 +51,7 @@ public int getCount() {
 
     @Override
     public Object getItem(int index) {
```

Comment:
Not sure if I understand: what is this "getItem" used for?

Reply:
Nothing, this method was used in Max's previous PRs, now it's never used.

Reply:
Can you remove it then completely?